pkgs/top-level/all-packages.nix

```diff
@@ -5863,6 +5863,8 @@ with pkgs;
 
   librenms = callPackage ../servers/monitoring/librenms { };
 
+  librenms-agent = callPackage ../servers/monitoring/librenms-agent { };
```

Comment:
Hint: move the default.nix file to pkgs/by-name/li/librenms-agent/package.nix and you can remove this line:

```suggestion
```

Reply:
@AndersonTorres , this is nice, I wasn't aware of it. :)